src/components/multisigParameters.vue

```diff
@@ -0,0 +1,70 @@
+<template>
+	<a-card title="Multisignature parameters" style="margin-top: 14px">
+		Addresses:
+		<li
+			v-for="ADDRESSES in mparams.addresses"
+			:key="ADDRESSES.address"
+			class="list_addresses"
+		>
+			<CheckCircleTwoTone v-if="ADDRESSES.signed" twoToneColor="LightGreen" />
```

Comment:
Is the value of `twoToneColor` mentioned in docs, or are we using our own colors..?, if it is own styling add the colors to constants

Reply:
the color we are using is in css.
I lookup from here https://www.w3schools.com/cssref/css_colors.asp

Reply:
So let's create a constant of it in `constants` file and then use it here, since in future there may be possibility we may require it at different places, that would be helpful in it